example/integration_tests/components-ce/sample/Main.java

```diff
@@ -1,5 +1,17 @@
+import org.graalvm.polyglot.Context;
+
+import java.io.BufferedOutputStream;
+import java.io.OutputStream;
+
+
 public class Main {
     public static void main(String args[]) {
-        System.out.println("Hello, GraalVM!");
+        OutputStream myOut = new BufferedOutputStream(System.out);
+        Context context = Context.newBuilder("js")
+                .out(myOut)
+                .allowAllAccess(true)
+                .build();
+        context.eval("js", "42");
+        context.close();
```

Comment:
i've kept your test code, but moved it to `components-ce`, where we are already testing components.